src/plugins/stats.ts

```diff
@@ -31,8 +31,8 @@ module.exports = async robot => {
     return github.paginate(req, res => res.data)
   }
 
-  async function popularInstallations (installations) {
-    let popular = await Promise.all(installations.map(async installation => {
+  async function popularInstallations (installations): Promise<Array<PopularType>> {
+    let popular: PopularType[] = await Promise.all<PopularType>(installations.map(async (installation) => {
```

Comment:
you don't need to add types here, if you defined a type for `installations` above, it will be inferred by TS, but it's still good for readability to set the type of the variables that you define, so you can leave `let popular: PopularType[]`, but you should remove `<PopularType>` since manually casting a function will hide errors that TS would find otherwise